contracts/MonthlySubscriptionPlan.sol

```diff
@@ -7,9 +7,7 @@ import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "./Subscriptions.sol";
 import "./ContractRegistry.sol";
 
-contract MonthlySubscriptionPlan is Ownable {
-
-    IContractRegistry contractRegistry;
+contract MonthlySubscriptionPlan is Ownable, ContractRegistryAccessor {
```

Comment:
there is a pitfall here - ContractRegistryAccessor is already an Ownable. this will lead to possible hiding of one of the owner address variables as they have the same name. The order in which they are specified determines which member will be hidden. the question is - will the owner variable used in ContractRegistryAccessor be the same value as the value observed from this contract after changing an owner. 

Maybe we should just omit Ownable here and avoid the question altogether. But it's nicer to be explicit about it...

Alternately, maybe check out Roles contracts do this in openzeppelin. lets discuss?

Reply:
I think I did that with the rest of the contracts, probably skipped that one. Will remove Ownable.